src/block.rs

```diff
@@ -1,7 +1,6 @@
-use crate::db_state::RowFeature;
 use crate::error::SlateDBError;
-use crate::row_codec::encode_row_v0;
-use crate::types::RowAttributes;
+use crate::row_codec::{SstRowCodecV1, SstRowEntry};
```

Comment:
It isn't part of this PR, but I think we should remove the:

```
    // The row level features enabled for this SST
    row_features: [SstRowFeature];
```

from `sst.fbs` and all the related code since we're no longer using row features and instead encoding them with each row.

Reply:
thank you for suggestion, removed it in 8ee5649